steering_controllers_library/test/steering_controllers_library_open_loop_timeout_params.yaml

```
@@ -0,0 +1,17 @@
+test_steering_controllers_library_open_loop_timeout:
```

```suggestion
test_steering_controllers_library:
```
this has to match the node name, which is hardcoded [here](https://github.com/ros-controls/ros2_controllers/blob/3850a08d73424963831a92bda2a9c332d8f0aa6f/steering_controllers_library/test/test_steering_controllers_library.hpp#L155)